Controller/SiteAdminController.php

```diff
@@ -63,4 +64,16 @@ public function snapshotsAction()
             'object' => $object,
         ));
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getRequest()
```

Comment:
Please make the method final. Not sure why we didn't do it in the piece of code you copied from.

Reply:
The class \Sonata\PageBundle\Controller\SiteAdminController extends the class \Sonata\AdminBundle\Controller\CRUDController. The class \Sonata\AdminBundle\Controller\CRUDController contains the getRequest() I have to copy. I do not know why I had to copy the method at all.

I will do the method final and try to improve my commit message.

Reply:
Oooooh I did not notice that. Ok then, just remove it, and sorry for the trouble!

Reply:
`CRUDController` is aliased to `Controller`, that's probably why I did not notice that.